libfwupdplugin/fu-common-version.c

```diff
@@ -545,3 +549,41 @@ fu_common_vercmp_full (const gchar *version_a,
 	}
 	return fu_common_vercmp_safe (version_a, version_b);
 }
+
+/**
+ * fu_common_version_kernel:
+ * @minimum_kernel: The minimum kernel version to check against
+ * @error: (nullable): optional return location for an error
+ *
+ * Determines if the system is running at least a certain required kernel version
+ *
+ * Since: 1.6.2
+ **/
+gboolean
+fu_common_version_kernel (const gchar *minimum_kernel, GError **error)
```

Comment:
I feel like this should have `check` or `verify` in the name somewhere. Maybe `fu_common_check_kernel_version` is more obvious?

Reply:
Sure, I'll shift that.  Should it live in this file or `fu-common.c` with such a change?

Reply:
Ahh, i missed that.  fu-common.c certainly, thanks.